test/metabase/test/data/sql.clj

```diff
@@ -285,11 +287,15 @@
 
 (defmethod add-fk-sql :sql/test-extensions
   [driver {:keys [database-name] :as dbdef} {:keys [table-name]} {dest-table-name :fk, field-name :field-name}]
-  (let [quot               #(sql.u/quote-name driver %1 (ddl.i/format-name driver %2))
-        dest-table-name    (name dest-table-name)
-        dest-table-pk-name (->> (get-tabledef dbdef dest-table-name)
-                                :field-definitions
-                                fielddefs->pk-field-name)]
+  (let [quot                #(sql.u/quote-name driver %1 (ddl.i/format-name driver %2))
+        dest-table-name     (name dest-table-name)
+        pk-names            (->> (get-tabledef dbdef dest-table-name)
+                                 :field-definitions
+                                 fielddefs->pk-field-names)
```

Comment:
nit: indentation is off here - it seems from `dest-table-pk-name` first being pluralized, and only later being de-prefixed to `pk-names`.